src/actions/actions.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios'
+
+export const getCenter='getCenter'
+export const getLocations='getLocations'
+export const reverseLocations='reverseLocations'
+
+export function fetchLocations() {
```

Comment:
add documentation to all the functions/actions in [JSDoc](https://devhints.io/jsdoc)